compiler/pipes/check-conversions.cpp

```diff
@@ -3,16 +3,12 @@
 // Distributed under the GPL v3 License, see LICENSE.notice.txt
 
 #include "compiler/pipes/check-conversions.h"
-
-#include "common/termformat/termformat.h"
-
```

Comment:
Please, leave one \n here (we separate including this-file.h from other includes)

Reply:
Fixed.